oauth2/src/main/java/com/okta/spring/boot/oauth/Okta.java

```diff
@@ -135,6 +130,61 @@ public static HttpSecurity configureOAuth2WithPkce(HttpSecurity http, ClientRegi
         return http;
     }
 
+    /**
+     * Configures the {@code http} with an OAuth2 Login, that sends an audience parameter with the authorize request.
+     *
+     * @param http                         the HttpSecurity to configure
+     * @param clientRegistrationRepository the repository bean, this should be injected into the calling method.
+     * @param properties                   the OktaOAuth2Properties bean, this should be injected into the calling method.
+     * @return the {@code http} to allow method chaining
+     */
+    public static ServerHttpSecurity configureOAuth2WithAudience(ServerHttpSecurity http,
+                                                                 ReactiveClientRegistrationRepository clientRegistrationRepository,
+                                                                 OktaOAuth2Properties properties) throws Exception {
+        // Don't add audience parameter if empty or default
+        if (isDefaultAudience(properties)) {
+            return http;
+        }
+        http.oauth2Login(oauth2 -> oauth2
+            .authorizationRequestResolver(
+                reactiveAuthzRequestResolver(clientRegistrationRepository, properties.getAudience())
+            )
+        );
+
+        return http;
+    }
+
+    private static boolean isDefaultAudience(OktaOAuth2Properties properties) {
+        String audience = properties.getAudience();
+        return audience == null || audience.isEmpty() || audience.equals("api://default");
```

Comment:
This means that `api://default` will never be sent if it's the audience. I tried to make it so it only happens when it's Auth0, but was unable to figure out how to do it.

Reply:
I guess it should not be a problem sending it for Okta too.